app/data/schema/schema-v1.json

```diff
@@ -151,6 +162,9 @@
     "theme": {
       "type":"string"
     },
+    "give_notice":{
+      "type": "boolean"
```

Comment:
There something the ONS calls this; there are actually multiple notices something like "legal basis" for running the survey and the legal basis for most business surveys is the statistics of trade act. Consider making this an enum of the various legal basis; starting with voluntary and stats of trade 1941? Jenny can probably provide the actual names etc.